lib/analytics/course_students_csv_builder.rb

```diff
@@ -42,6 +71,8 @@ def row(courses_user)
     row << courses_user.character_sum_draft
     row << courses_user.references_count
     row << newbie?(courses_user.user)
+    row << @created_articles[courses_user.user_id]
+    row << @edited_articles[courses_user.user_id]
```

Comment:
rubocop is complaining the size of this method is too high 

Metrics/AbcSize: Assignment Branch Condition size for row is too high. [<1, 26, 0> 26.02/23]

I suppose I should supress this somehow or just ignore?

Reply:
yeah, you can suppress it by putting comments before and after the method:

`# rubocop:disable Metrics/AbcSize`
`# rubocop:enable Metrics/AbcSize`

For a method like this where it's following a very clear pattern many times, that complexity metric isn't a useful indicator that the code is going to be hard to understand or change.